webgrid/__init__.py

```diff
@@ -852,7 +864,7 @@ def apply_validator(self, validator, value, qs_arg_key):
             return None
 
     def set_export_to(self, to):
-        if to in ('xls',):
+        if to in ('xls', 'xlsx'):
```

Comment:
Please create an attribute on the grid named something like `allowed_export_targets` for this tuple.